Consume attestation files (#706)
anchore/grype: README.md

grype --add-cpes-if-none --distro alpine:3.10 sbom:some-apline-3.10.spdx.json
```

### Scan attestations

> Grype can scan SBOMs from attestations, as long as they are encoded as part of [in-toto envelope](https://github.com/in-toto/attestation/blob/main/spec/README.md#envelope).

Reviewer: nit: I think we remove comma here

Reviewer: Apologies if this came from my earlier suggestion

Reviewer: Comment from meeting - Add information to feature session for this PR

Reviewer: Comment from meeting - update scheme section to talk about new source and how it is a more secure version of the grype workflow

Reviewer: Comment from meeting - update Scan attestations section Examples with full connective story of commands

> Grype can scan SBOMs from attestations, as long as they are encoded as part of (in-toto envelope)[https://github.com/in-toto/attestation/blob/main/spec/README.md#envelope], that is the format used by Syft.

Reviewer: nit:
```suggestion
Grype can scan SBOMs from attestations, as long as they are encoded as part of and [in-toto envelope](https://github.com/in-toto/attestation/blob/main/spec/README.md#envelope).
```

This should clean up the link. Let me know if you think it reads better by dropping the end statement
